server/channels/store/storetest/settings.go

```diff
@@ -69,7 +54,6 @@ func MySQLSettings(withReplica bool) *model.SqlSettings {
 	dsn := os.Getenv("TEST_DATABASE_MYSQL_DSN")
 	if dsn == "" {
 		dsn = getDefaultMysqlDSN()
-		mlog.Info("No TEST_DATABASE_MYSQL_DSN override, using default", mlog.String("default_dsn", dsn))
```

Comment:
The "default" in this case isn't even relevant, since we go on to generate a random database. Stop logging this unless we are using the env var in the first place.